src/components/Login/index.js

```diff
@@ -40,6 +42,8 @@ function Login() {
             res.json().then(data => {
                 setTokens(data.access_token, data.refresh_token);
                 console.log('data is :', data);
+                console.log(jwt(data.access_token));
```

Comment:
We shouldn't leave console logs behind. Try and remove console logs before committing code